pkgs/development/cuda-modules/packages/nccl-tests.nix

```diff
@@ -34,49 +40,74 @@ backendStdenv.mkDerivation (finalAttrs: {
   };
```

Comment:
TODO: All FODs in `cudaPackages` should provide `name`, which must depend on version or something else which forces invalidation.

Also, switch to use of `tag`.